### PR TITLE
New Loaders

### PR DESCRIPTION
**Pull request by theblindspring**
This pull request addresses issue #1194.

### CODE REVIEW

File: app/views/helpers/lists.php [hunk 1]
```diff
@@ -470,7 +470,7 @@ public function displayAddSentenceForm($listId)
                 'class' => 'submit'
             )
         );
-        echo "<div class='loader-container'></div>";
+        echo "<md-progress-circular id='list_add_loader' md-diameter='11px' style='display: none;'></md-progress-circular>";
```
Comment (trang) [inline app/views/helpers/lists.php]: `md-diameter` can be increased to 16px. 11px is a bit too small.
Also note that you don't need to specify "px". `md-diameter="16"` works too.

File: app/views/helpers/menu.php [hunk 2]
```diff
@@ -645,10 +647,7 @@ public function displayMenu(
 
         <li>
         <?php
-        echo $this->Html->div('loader-small loader', '', array(
-            'id' => '_'.$sentenceId.'_in_process',
-            'style' => 'display:none',
-        ));
+        echo "<md-progress-circular  md-diameter='11px' style='display: none;' id='_" . $sentenceId ."_in_process'></md-progress-circular>";
```
Comment (trang) [inline app/views/helpers/menu.php]: The loaders up there for the favorites and correctness can be removed, and you can use this `#_{sentenceId}_in process` loader instead.

File: app/views/helpers/sentences.php [hunk 3]
```diff
@@ -553,6 +550,7 @@ public function displayGenericSentence(
         if( $isFavoritePage && $this->params['pass'][0] == CurrentUser::get('username') ){
             echo '<div class="favorite-page column">';
             $this->Menu->favoriteButton($sentenceId, true, true, true);
+            echo '<li class="option favorite progress" style="display: none;"><md-progress-circular md-diameter="24px"></md-progress-circular></li>';
```
Comment (trang) [inline app/views/helpers/sentences.php]: This loader doesn't align very well, but I think to fix the alignment we'd need to review the HTML of the whole `displayGenericSentence()` function, so we can leave it like that for now.

File: app/webroot/js/sentences.show_another.js [hunk 4]
```diff
@@ -16,8 +16,9 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-
+$("#random-progress").hide();
```
Comment (trang) [inline app/webroot/js/sentences.show_another.js]: This looks strange as it's outside of the `$(document).ready()`. What is for?

File: app/views/helpers/menu.php (hunk 2 above)
Comment (theblindspring) [inline app/views/helpers/menu.php]: I am not sure what code you are talking about? Are you saying I should remove the 'loader-small' class from 'ul' loader?

Comment (theblindspring) [inline app/views/helpers/menu.php]: Okay, so you are saying reuse the loader used for adding a sentence to a list?

Comment (trang) [inline app/views/helpers/menu.php]: Yes.

File: app/views/helpers/lists.php (hunk 1 above)
Comment (theblindspring) [inline app/views/helpers/lists.php]: Should I change just these instances or all instances of 11 to 16?

Comment (trang) [inline app/views/helpers/lists.php]: All instances. Unless it breaks the UI, in which case, just make it as close as possible to 16px.

**Pull request merged**